hsctl

```diff
@@ -243,18 +244,23 @@ loaddb_hs() {
 
 migrate_hs() {
     echo "MIGRATE:"
-    echo "  - docker exec -u hydro-service hydroshare python manage.py migrate sites --noinput"
-    docker exec -u hydro-service hydroshare python manage.py migrate sites --noinput
-    echo "  - docker exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput"
-    docker exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput
-    echo "  - docker exec -u hydro-service hydroshare python manage.py fix_permissions"
-    docker exec -u hydro-service hydroshare python manage.py fix_permissions
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate sites --noinput"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate sites --noinput
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate --fake-initial --noinput"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py migrate --fake-initial --noinput
+    echo "  - docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py fix_permissions"
+    docker exec -u hydro-service hydroshare python -W ignore::DeprecationWarning:: manage.py fix_permissions
 }
 
 manage_py_hs() {
     echo "MANAGEPY:"
-    echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}"
-    docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}
+    if [ "$2" == "--verbose" ]; then
+        echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:3}"
+        docker exec -u hydro-service -ti hydroshare python manage.py ${@:3}
+    else
+        echo "  - docker exec -u hydro-service -ti hydroshare python -W ignore::DeprecationWarning:: manage.py ${@:2}"
```

Comment:
going along with my other comment on passing in arguments, we could write the arguments we're passing to docker in a bash variable as a string and append any arguments instead of doing an if/else with the different arguments.
i.e.
```
docker_args = ""
case $ in
  verbose) docker_args = docker_args + " -W ignore::DeprecationWarning::"

echo "  - docker exec -u hydro-service -ti hydroshare python $docker_args  manage.py ${@:2}"
```
My bash might be a bit rusty.

Reply:
oh, this is just an echo, but you get the idea.

Reply:
That looks nice, yeah

Reply:
looking at it now, looks like I got the case wrong too.   You don't want those arguments if it's verbose... whoops, but you already knew that.